cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -259,13 +259,19 @@ include_directories(SYSTEM ${THRIFT_INCLUDE_DIR} ${THRIFT_INCLUDE_DIR}/thrift)
 message(STATUS "Thrift include dir: ${THRIFT_INCLUDE_DIR}")
 message(STATUS "Thrift static library: ${THRIFT_STATIC_LIB}")
 message(STATUS "Thrift compiler: ${THRIFT_COMPILER}")
+message(STATUS "Thrift version: ${THRIFT_VERSION}")
```

Comment:
Should this be `THRIFT_VENDOR_VERSION`?

Reply:
No, this variable is set by `FindThrift.cmake`